core.js

```diff
@@ -3913,7 +3913,7 @@ const _makeWindow = (options = {}, parent = null, top = null) => {
       }
     }
   };
-  window.requestAnimationFrame = _makeRequestAnimationFrame(window);
+  window.requestAnimationFrame = _makeRequestAnimationFrame(window, 2);
```

Comment:
Can this be rebased to zero? So the user raf is zero and anything before that is negative?

Reply:
Not easily. The user RAF has to come after the device RAF, and rebasing the user RAF to zero means either defaulting the parameter to a negative value or passing in a specific constant in N different places. (Where N is about 4.)

Reply:
Hm, I really don't like using these constants in general. It makes it essentially impossible to use the raf sorting feature if the core code is defining its own constants.

I'd prefer to either get rid of rafs inside of this code, or define basis constant for the frame times so user code can at least do a `+`/`-` off of them.

Reply:
Sure, think over a design that makes sense and let me know what you decide.